cypress/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Does this config (also) adhere to the rules in the [default config](https://github.com/colonial-heritage/dataset-browser/blob/main/tsconfig.json)? Or doesn't this matter?

Reply:
I now extend the default config, and only kept the changes needed for Cypress.